extensions-core/s3-extensions/src/main/java/org/apache/druid/data/input/s3/S3InputSourceConfig.java

```diff
@@ -90,6 +91,24 @@ public boolean isCredentialsConfigured()
            secretAccessKey != null;
   }
 
+  @JsonIgnore
+  public boolean isAssumeRoleArnConfigured()
+  {
+    return assumeRoleArn != null && !assumeRoleArn.trim().isEmpty();
```

Comment:
I donot think a trim is required here and below. 
If a user sets a wrong property its a wrong property even if its just a space at the start or the end.

Reply:
@cryptoe ok, if that's Druid's style, no problem. I got used to doing that as part of my own convention.

Reply:
@cryptoe done.